src/docbuilder/rustwide_builder.rs

```diff
@@ -427,21 +427,23 @@ impl RustwideBuilder {
                     add_doc_coverage(&mut conn, release_id, doc_coverage)?;
                 }
 
-                add_build_into_database(&mut conn, release_id, &res.result)?;
+                let build_id = add_build_into_database(&mut conn, release_id, &res.result)?;
+                let build_log_path = format!("build-logs/{}/{}.txt", build_id, default_target);
+                self.storage.store(build_log_path, res.build_log)?;
```

Comment:
Hmmm, my plan was to delete the column from the database post-migration. Do we expect to ever have this actually fail?

Reply:
Well, I think we should treat it the same as failing to upload documentation. And looking at the code we ... give up on the first failure :facepalm: so it could be inconsistent at any point if the builder loses internet for some reason.

So I think there's room for improvement here, but it's probably a large change and shouldn't block this PR.

Reply:
I actually ended up changing this in my next steps preparing to support all the logs, I think we can capture almost any error that happens during the build and attempt to shove it into the database.